language/stdlib/modules/script_whitelist.move

```diff
@@ -0,0 +1,16 @@
+address 0x0:
+
+module ScriptWhitelist {
+    use 0x0::LibraConfig;
+
+    struct T { payload: vector<u8> }
+
+    public fun initialize(payload: vector<u8>) {
+        // Only callable by the Association address for now.
```

Comment:
This comment looks out of place given that  the check is in the callee. Kill it?